src/System.Net.Mail/src/System/Net/Mail/SmtpClient.cs

```diff
@@ -52,6 +52,7 @@ public class SmtpClient : IDisposable
         private ContextAwareResult _operationCompletedResult = null;
         private AsyncOperation _asyncOp = null;
         private static AsyncCallback s_contextSafeCompleteCallback = new AsyncCallback(ContextSafeCompleteCallback);
+        private static NetworkCredential s_defaultNetworkCredential = CredentialCache.DefaultNetworkCredentials;
```

Comment:
Was using the property in multiple places, so put it in a field. And CredentialCache is checking environment permissions etc, is caching harmful here?